### ISSUE TITLE
Print Visibility triggers anomaly when ssreflect is loaded.

### ISSUE THREAD

**Issue by jhjourdan** [coq/coq#10750]
On current master:

```coq
Require Import ssreflect.

Print Visibility.
(* uncaught Not_found *)
```

**Comment by SkySkimmer** [comment]
Since this is not specific to coqtop I edited the coqtop stuff away

**Comment by MSoegtropIMC** [comment]
I set the milestone to 8.11 because 8.11 beta has compatibility issues between ssreflect and the standard library and this could help to debug such issues. See e.g. the discussion in #11212.

**Comment by herbelin** [comment]
This is actually not related to importing ssreflect. This is only related to the presence of only-printing notations. A minimal fix is available at #11276.
